content/blog/cloudnative-pg-1-23.2-released/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "CloudNativePG 1.23.2, 1.22.4 and 1.21.6 Released!"
+date: 2024-06-12T23:49:11+02:00
+draft: false
+image:
+    url: 58364dde1adc4a0a8.37186404-2048x1445.jpg
+    attribution: from <a href="https://wordpress.org/photos/photo/58364dde1a/">Saurabh</a>
+author: gbartolini
+tags:
+ - release
+ - postgresql
+ - postgres
+ - kubernetes
+ - k8s
+ - cloudnativepg
+summary: The CloudNativePG community has released a new update for the supported 1.23, 1.22 and 1.21 versions of the CloudNativePG operator. Version 1.21 has reached End-of-Life (EOL).
+---
+The **CloudNativePG Community** is thrilled to announce a new update for the
+**CloudNativePG Operator**, now available for the supported versions 1.23,
+1.22, and 1.21.
+
+Thanks to comprehensive refactoring and enhancement activities, we've
+significantly improved the reliability of our automated End-to-End (E2E) tests.
+With contributions from [EDB](https://enterprisedb.com), we've tested the
+operator on all major cloud providers, identified several defects, and resolved
+them in these new releases, ensuring a more stable and resilient software.
+
+**Versions 1.23.2, 1.22.4, and 1.21.6** are patch releases that contain crucial
+bug fixes, addressing issues such as:
+
+- Unnecessary reloads and shutdowns of PostgreSQL
+- Excessive CPU usage due to unnecessary loops
```

Comment:
perhaps we could say that this happened on fenced instances, otherwise I'd specify a bit what those "unnecessary loops" were

Reply:
```suggestion
- Excessive CPU usage on fenced instances, due to busy waiting loop
```